packages/browser/src/client.ts

```diff
@@ -122,4 +134,32 @@ export class BrowserClient extends BaseClient<BrowserClientOptions> {
     }
     super._sendEvent(event);
   }
+
+  /**
+   * Sends client reports as an envelope.
+   */
+  private _flushOutcomes(): void {
+    const outcomes = this._clearOutcomes();
+
+    if (outcomes.length === 0) {
+      IS_DEBUG_BUILD && logger.log('No outcomes to send');
+      return;
+    }
+
+    if (!this._dsn) {
+      IS_DEBUG_BUILD && logger.log('No dsn provided, will not send outcomes');
+      return;
+    }
+
+    IS_DEBUG_BUILD && logger.log(`Sending outcomes:\n${JSON.stringify(outcomes, null, 2)}`);
+
+    const url = getEnvelopeEndpointWithUrlEncodedAuth(this._dsn, this._options.tunnel);
+    const envelope = createClientReportEnvelope(outcomes, this._options.tunnel && dsnToString(this._dsn));
```

Comment:
It was [like this](https://cs.github.com/getsentry/sentry-javascript/blob/4e722eb8778e27d7910e96ccb1aac108bcbea146/packages/browser/src/transports/base.ts#L145) before but I'm very unsure whether the `&&` in this line should actually be a `||`.